include/multipass/cli/client_common.h

```diff
@@ -55,14 +55,12 @@ bool update_available(const UpdateInfo& update_info);
 std::string update_notice(const multipass::UpdateInfo& update_info);
 
 template <typename Request, typename Reply>
-void handle_user_password(grpc::ClientReaderWriterInterface<Request, Reply>* client, Terminal* term)
+void handle_password(grpc::ClientReaderWriterInterface<Request, Reply>* client, Terminal* term)
 {
     Request request;
-    const auto [username, password] = MP_CLIENT_PLATFORM.get_user_password(term);
-
-    request.mutable_user_credentials()->set_username(username);
-    request.mutable_user_credentials()->set_password(password);
+    const auto password = MP_CLIENT_PLATFORM.get_password(term);
 
+    request.set_password(password);
```

Comment:
Since we don't use `password anywhere else in this scope and it's a single variable, you could do instead:

```suggestion
    request.set_password(MP_CLIENT_PLATFORM.get_password(term));
```